src/components/mgt-person/mgt-person.ts

```diff
@@ -177,7 +210,7 @@ export class MgtPerson extends MgtTemplatedComponent {
    * @memberof MgtPerson
    */
   public disconnectedCallback() {
-    window.removeEventListener('click', this.handleWindowClick);
+    window.removeEventListener('click', e => this.handleWindowClick(e));
```

Comment:
This won't work because the arrow function is defining a new instance which will not match the instance from `addEventListener` above.

Reply:
Doh! Fixed